torch/_inductor/graph.py

```diff
@@ -1946,6 +1947,7 @@ def _compile_to_module(self) -> ModuleType:
 
         inductor_meta = autotune_cache.inductor_meta_from_config()
         AutotuneCacheBundler.begin_compile(inductor_meta, code=code)
+        TritonBundler.begin_compile()
```

Comment:
Kind of wondering why this doesn't live in load_by_key_path (which is what actually triggers Triton compiles)

Reply:
load_by_key_path is called right after this. Do you want this to be inside load_by_key_path? The other caller of load_by_key_path in codecache does not need this as we are not going to be writing to cache after that

Reply:
Well, I'm on the record for saying that load_by_key_path being called from two sites (once from here, once from the cache hit site) has been the source of endless confusion (e.g., Triton compile time on cache hit getting incorrectly attributed to inductor phase instead of code_gen phase). I guess if you want @masnesral to fix it all in one go that's fine. Not sure if Sam has an opinion.